platform-common-core/src/main/java/com/softicar/platform/common/core/number/formatter/BigDecimalScaleApplier.java

```diff
@@ -0,0 +1,92 @@
+package com.softicar.platform.common.core.number.formatter;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+/**
+ * Applies a given scale to {@link BigDecimal} values.
```

Comment:
```suggestion
 * Applies a given scale to a {@link BigDecimal} value.
```
...since this whole class is built around modifications of a single value.

Reply:
actually, in this case, the class instance can be used multiple times, i wonder if I should change BigDecimalFormatter in a similar way, so it can be configured an used for multiple values?

Reply:
- not sure if the exact wording is so important here, but i changed it, as you wished
- changed BigDecimalFormatter